### PR TITLE
Sequenza with cram support and Blacklist filtering

### PR DESCRIPTION
**Pull request by Kdreval**
This update for the existing sequenza module includes support of cram files, filtering of variants that fall within blacklisted regions, and harmonizing the module with recent updates (oncopipe version check, absolute symlinks for inputs, `in_module` symlinking for outputs, and resource unpacking). I have also removed the default scratch directory from config.
I have edited v1.4 files to ensure easy view of the changes that I implemented. Once approved, I will bump this to version 1.5 and restore v1.4 back.

### CODE REVIEW

**Review by lkhilton — DISMISSED** [review]
Looks great. Does this really need a version bump though? I don't think I see anything that would break existing sequenza setups except for the way resources are unpacked.

**Comment by Kdreval** [comment]
Thank you! The only new feature that is added here is filtering towards the encode blacklist. It will not break the existing outputs and pipeline. I am thinking to do version bump so we can distinguish whether it was filtered only towards dbsnp, or blacklist ad well.

**Comment by rdmorin** [comment]
I agree with Laura that a version bump may not be warranted. If the rationale is to increase it so we can distinguish it in GAMBL then why not just change the config and update the parent directory to explicitly refer to the change in filtering?

**Comment by Kdreval** [comment]
Thank you all for the comments! I agree that the update is minor and a version update is not required. I only changed the CHANGELOG to keep the record of added features, but associate it with v1.4. This dismissed the approval, so will need another one.

File: modules/sequenza/1.4/src/bash/filter_seqz.sh [hunk 1]
```diff
@@ -18,11 +18,19 @@ set -euf -o pipefail
 zcat "${SEQZ_FILE}" \
 	| egrep -v "^chromosome" \
 	| awk 'BEGIN {FS="\t"} $9 == "het" {print $1 ":" $2}' \
-	| sort -S "${BUFFER_SIZE}" --parallel 10 \
+	| sort -S "${BUFFER_SIZE}" --parallel 8 \
```
Comment (rdmorin) [inline modules/sequenza/1.4/src/bash/filter_seqz.sh:21]: In theory the 8 should be a parameter that is passed in from Snakemake (ideally)

**Review by rdmorin — APPROVED** [review]
Up to you if you want to change that script. I can live with this version since that issue has been there from the start.

**Comment by rdmorin** [comment]
Should this be merged, @Kdreval ?

**Comment by Kdreval** [comment]
Yes. I thought to reorganize it to remove checkpoint, but this can be done later for future versions

**Pull request merged**